frontend/src/js/small-tab-navigation/HoverNavigatable.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+import { useDrop } from "react-dnd";
+import { DNDType } from "../common/constants/dndTypes";
+
+
+interface PropsT {
+    timeUntilNavigateOverride?: number;
```

Comment:
Let's not add props that we don't use. Even though it's a nice thought for the future, if we're never passing `timeUntilNavigateOverride` anywhere, let's remove the prop